code/modules/clothing/under/miscellaneous.dm

```diff
@@ -254,6 +254,78 @@ obj/item/clothing/under/genericr
 	style = STYLE_HIGH
 	price_tag = 300
 
+//stylish vendor clothes
```

Comment:
are you seriously so lazy to add style = style_high that you made them all a child of this?

bad practice. no.

Reply:
> are you seriously so lazy to add style = style_high that you made them all a child of this?

yes

~~gonna change that, literally 1984~~

Reply:
done